docs/js/viz/index.md

```diff
@@ -13,9 +13,23 @@ Once this is correctly installed, you can start creating visualizations, for exa
 
 ## Creating visualizations
 
-To create a viz, you must specify a **query**, **container** and **options**.  For example, to [visualize a chart](#chart-visualizations):
+To create a viz, you must specify a **query** to visualize, a **viz type** to display it as and a **container** to display it inside. You can also supply **options** to specify how the viz will be configured.
```

Comment:
To create a viz, you must:
* specify a **query** that contains the data to visualize
* a **viz type**
* a **container**
You can also supply **options** to configure the appearance of the viz.

Also, link the **emphasised** terms to relevant sections in the doco